### PR TITLE
Fix/fix input size

### LINKED ISSUE #958: [Bug] The fields in the API pane don't revert to their original size when they lose focus
## Steps to reproduce

1. Go to the API pane in the Appsmith editor
2. Type a very long URL in the URL field. Optionally, type a long text in the input fields for query params or header params.

## Observation
The fields expand vertically to show the complete text when it's in focus. When the input field loses focus, it doesn't go back to it's original size.

## Expected behaviour
When an input field loses focus, it should go back to it's original size.

### PR DESCRIPTION
**Pull request by ThakurKarthik**
## Description
fix extra padding issue of input field after losing focus.

Fixes #958
> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce. 
> Please also list any relevant details for your test configuration.

- Verified in different input fields in local

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

Screenshot from 2020-10-15 01-05-52

### After losing focus

Screenshot from 2020-10-15 01-06-11

### CODE REVIEW

**Comment by ThakurKarthik** [comment]
@hetunandu @Nikhil-Nandagopal I tried to remove the padding-bottom from CodeMirror-Scroll class by setting to 0 then i noticed vertical scrollbars coming up in input fields after losing focus,so i searched for this an came across `scrollbarStyle: string` prop by setting this to "null" it seems to remove that padding without changing any CSS. Can you verify this and see if it is breaking any other areas?

Thanks

File: app/client/src/components/editorComponents/CodeEditor/index.tsx [hunk 1]
```diff
@@ -118,6 +118,7 @@ class CodeEditor extends Component<Props, State> {
         lineWrapping: this.props.size !== EditorSize.COMPACT,
         lineNumbers: this.props.showLineNumbers,
         addModeClass: true,
+        scrollbarStyle: "null",
```
Comment (hetunandu) [inline app/client/src/components/editorComponents/CodeEditor/index.tsx:121]: Can we add this prop conditionally when EditorSize.COMPACT is true?
Similar to lineWrapping prop

**Review by hetunandu — CHANGES_REQUESTED** [review]
Small change

**Comment by ThakurKarthik** [comment]
@hetunandu can you review once, Thanks !

**Review by hetunandu — APPROVED** [review]

**Pull request merged**